src/openrct2/common.h

```diff
@@ -135,10 +135,17 @@ using money64 = fixed64_1dp;
 #define FIXED_1DP(whole, fraction) FIXED_XDP(1, whole, fraction)
 #define FIXED_2DP(whole, fraction) FIXED_XDP(10, whole, fraction)
 
-// Construct a money value in the format MONEY(10,70) to represent 10.70. Fractional part must be two digits.
-#define MONEY(whole, fraction) ((whole)*10 + ((fraction) / 10))
+// Convert money literal to money32
+constexpr money32 operator"" _GBP(long double money)
+{
+    return money * 10;
+}
+// Helper function to convert runtime values to money
+constexpr auto MONEY(int32_t whole, int32_t fraction)
```

Comment:
Is this function still used?

Reply:
I would like to see it gone

Reply:
@duncanspumpkin It is still used in InteractiveConsole.cpp and LandSetHeightAction.cpp
![image](https://user-images.githubusercontent.com/38046772/109684160-3bb6f680-7b3d-11eb-9c3a-d1b946c6280a.png)

Reply:
And can they be modified to not require it?

Reply:
I've gone ahead and done that, it wasn't as difficult as I thought.